safeframe/jac/readme.md

```diff
@@ -608,6 +608,34 @@ function initSpotimMessageHandlers() {
     });
   }
 
+  function subscribeToClipboardWrite() {
```

Comment:
above the implementation proposal there is a detailed explanation on each feature separately, can you add this to there as well?